catt/controllers.py

```diff
@@ -146,42 +147,36 @@ def __init__(self, device_name):
 
         self.cast.wait()
 
-    def _check_state(self):
-        if self.cast.app_id == "E8C28D3C" or not self.cast.app_id:
-            raise CattCastError("Chromecast is inactive.")
-        self.cast.media_controller.block_until_active(1.0)
-        if self.cast.media_controller.status.player_state in ["UNKNOWN", "IDLE"]:
-            raise CattCastError("Nothing is currently playing.")
+        if state_check:
```

Comment:
I can change it back if you prefer, just let me know.
Another thing: In cli we have a `stop` command which is calling `CastController.kill`, and then there is a `CastController.stop` which is currently unused. Do you want to adress that?